### ISSUE TITLE
Poor result on kaggle 2016 ultrasound nerve dataset

### ISSUE THREAD

**Issue by GWwangshuo** [ternaus/robot-surgery-segmentation#9]
I am trying to change surgery dataset to  custom dataset, the kaggle ultrasound nerve dataset, which can be found in https://www.kaggle.com/c/ultrasound-nerve-segmentation/data. However, the result is far less than the starting point Keras version UNet here, https://github.com/jocicmarko/ultrasound-nerve-segmentation.
 The training images and corresponding masks in this custom dataset are both one channel with the size 420, 580. I have cropped them to 416, 576 and use the modified RoboticsDataset to load them. 
![78051530838794_ pic](https://user-images.githubusercontent.com/6948631/42354659-eccad648-80fa-11e8-97ed-344d621ed06a.jpg)
each subdataset consists
![78071530838916_ pic](https://user-images.githubusercontent.com/6948631/42354703-384e8768-80fb-11e8-9e44-a9018f0671d5.jpg)
In images folder, there are 1127 images.

Training hyperparameters:
--batch size 16
--learing rate 0.00001
--epoch 100
--Adam
--UNet

Final IoU results: (%)
-- 17.31

The IoU achieve 0.4 by using previous Keras version UNet. Could your offer me some help to illustrate this problem? Thanks a lot.

**Comment by ternaus** [comment]
I remember that competition, I finished 10th in it.

The metric was noisy, validation did not work well and post-processing added a lot.

But IOU = 17.31 looks pretty strange.  I would expect ~0.5-0.6 out of the box.

Did your train loss stop decreasing?

**Comment by GWwangshuo** [comment]
it seems the loss has been converged to 0.9 ~ 1.0. I have try to modify the
loss function to dice loss, but got the same problem. Since the ultrasound
nerve dataset consists images with one channel, robot surgery dataset with
three channels, the preprocess (Normalize) will be different. Will this
preprocessing be a problem? Besides, I have tried to modify RoboticsDataset
to support loading with one channel image. with 36 epochs, IoU = 0.48447
which is far from 10 epochs IoU = 0.753 received in three channel input.
Any suggestions to deal with gray images input? Thanks a lot. I really
appreciate your help.
